gitignore

```diff
@@ -222,4 +222,4 @@ AppPackages
 project.lock.json
 msbuild.binlog
 *.project.lock.json
-/build/tools
+/build/tools
```

Comment:
The gitignore file seems to be now renamed. Can you rename it back to .gitignore?

Reply:
I spoke with @Mimetis and found that the issue is arising because of the Manipulation events. I am working on the fix.
Will update the gitignore file name as well.